backend/infrahub/core/query/node.py

```diff
@@ -117,8 +117,7 @@ def __init__(
 
 class NodeCreateAllQuery(NodeQuery):
     name = "node_create_all"
-
-    type: QueryType = QueryType.WRITE
+    type = QueryType.WRITE
```

Comment:
```suggestion

```
`QueryType.WRITE` is already default value

Reply:
I removed the default value altogether, I think it's best if each query is explicit about it